src/symbols.js

```diff
@@ -600,14 +614,23 @@ defineSymbol(math, main, mathord, "\u0131", "\\imath");
 defineSymbol(math, main, mathord, "\u0237", "\\jmath");
 
 defineSymbol(text, main, textord, "\u2013", "--");
+defineSymbol(text, main, textord, "\u2013", "\\textendash");
 defineSymbol(text, main, textord, "\u2014", "---");
+defineSymbol(text, main, textord, "\u2014", "\\textemdash");
 defineSymbol(text, main, textord, "\u2018", "`");
+defineSymbol(text, main, textord, "\u2018", "\\textquoteleft");
 defineSymbol(text, main, textord, "\u2019", "'");
+defineSymbol(text, main, textord, "\u2019", "\\textquoteright");
 defineSymbol(text, main, textord, "\u201c", "``");
+defineSymbol(text, main, textord, "\u201c", "\\textquotedblleft");
 defineSymbol(text, main, textord, "\u201d", "''");
+defineSymbol(text, main, textord, "\u201d", "\\textquotedblright");
 defineSymbol(math, main, textord, "\u00b0", "\\degree");
 defineSymbol(text, main, textord, "\u00b0", "\\degree");
 defineSymbol(math, main, mathord, "\u00a3", "\\pounds");
+defineSymbol(math, main, mathord, "\u00a3", "\\mathsterling");
+defineSymbol(text, main, mathord, "\u00a3", "\\pounds");
+defineSymbol(text, main, mathord, "\u00a3", "\\textsterling");
```

Comment:
Added.  Although I found [this discussion](https://gitter.im/Khan/KaTeX/archives/2014/09/24) where @xymostech suggests that maybe we should move away from mathord vs. textord, and rather just specify which font to use.  It seems LaTeX has a notion of "mathord" but no notion of "textord" (at least I've never seen one in debugging output)?